dev/Dockerfile

```diff
@@ -21,8 +21,6 @@ RUN apk --no-cache add socat
 WORKDIR "$SIA_DIR"
 COPY --from=builder /go/bin/siad ./
 COPY --from=builder go/bin/siac ./
-ENTRYPOINT socat tcp-listen:9980,reuseaddr,fork tcp:localhost:8000 & \
-  ./siad \
-    --modules "$SIA_MODULES" \
-    --sia-directory "$SIA_DATA_DIR" \
-    --api-addr "localhost:8000"
+COPY run.sh ./
```

Comment:
Should this be `dev/run.sh`? (Although it would be nice if we only had one `run.sh` file, looks like we introduce two identical versions here.)

Reply:
This is a problem that we need to address more globally, I think. There is an outstanding PR which adds an Alpine-based Sia image. That image will live under `/alpine` and will suffer from the same problem. That's why I chose to duplicate rather than reuse the dev version.

I'll look into ways to solve this but that can be done in a follow-up.